src/test/java/com/code_intelligence/jazzer/mutation/mutator/lang/EnumMutatorTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.code_intelligence.jazzer.mutation.mutator.lang;
+
+import static com.code_intelligence.jazzer.mutation.support.TestSupport.mockPseudoRandom;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.code_intelligence.jazzer.mutation.annotation.NotNull;
+import com.code_intelligence.jazzer.mutation.api.SerializingMutator;
+import com.code_intelligence.jazzer.mutation.support.TestSupport.MockPseudoRandom;
+import com.code_intelligence.jazzer.mutation.support.TypeHolder;
+import com.google.errorprone.annotations.Immutable;
+import org.junit.jupiter.api.Test;
+
+class EnumMutatorTest {
+  enum TestEnumOne { A }
+
+  enum TestEnum { A, B, C }
+
+  @Test
+  void testBoxed() {
+    SerializingMutator<TestEnum> mutator =
+        (SerializingMutator<TestEnum>) LangMutators.newFactory().createOrThrow(
+            new TypeHolder<@NotNull TestEnum>() {}.annotatedType());
+    assertThat(mutator.toString()).isEqualTo("TestEnum");
```

Comment:
It could be helpful to explicitly state that this is an `Enum` mutator, as the class name could be misleading. How do you like `Enum<TestEnum>`?

Reply:
It's much better, I added it, thanks :smile: